pkg/server/config.go

```diff
@@ -80,3 +128,22 @@ func (cfg *HandlerConfig) ToFlags(set *cli.FlagSet) *cli.FlagSet {
 
 	return set
 }
+
+func (cfg *MappingHandlerConfig) ToFlags(set *cli.FlagSet) *cli.FlagSet {
+	cfg.HandlerConfig.ToFlags(set)
+
+	f := set.NewSection("MAPPING SERVER OPTIONS")
+
+	f.StringVar(&cli.StringVar{
+		Name:    "default-resource-scope",
+		Target:  &cfg.DefaultResourceScope,
+		EnvVar:  "PMAP_MAPPING_RESOURCE_SCOPE",
+		Example: "{projects/test-project-id}",
+		Usage: `The default scope to search for resources. Format: 
+		projects/{PROJECT_ID}
+		projects/{PROJECT_NUMBER}
+		folders/{FOLDER_NUMBER}
+		organizations/{ORGANIZATION_NUMBER}`,
```

Comment:
Define `allowedScopes` in the file scope so you can use it here as well. Again, you usage of ` is wrong.